### PR TITLE
fix(VerticalTabs): set tab text to wrap when necessary

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
affects: patternfly3-react-lerna-root, patternfly-react-extensions

Changed the tab from a button to an anchor and set the width to be 100% of the li and to wrap on
long text.

ISSUES CLOSED: #663

**Link to Storybook**:
https://rawgit.com/jeff-phillips-18/patternfly-react/vertical-tabs-storybook/index.html?selectedKind=patternfly-react-extensions%2FWidgets%2FVertical%20Tabs&selectedStory=Vertical%20Tabs

### CODE REVIEW

File: packages/patternfly-3/patternfly-react-extensions/less/vertical-tabs.less [hunk 1]
```diff
@@ -7,21 +7,31 @@
...
   position: relative;
 
-  .btn.btn-link {
+  a {
     color: @vertical-tab-pf-color;
+    display: inline-block;
     font-size: 13px;
     padding-left: 15px;
+    width: 100%;
+    word-break: break-word;
```
Comment (rhamilto) [inline packages/patternfly-3/patternfly-react-extensions/less/vertical-tabs.less]: This should be optional but enabled by default?

Comment (jeff-phillips-18) [reply packages/patternfly-3/patternfly-react-extensions/less/vertical-tabs.less]: As in a parameter to the tab for wrap or ellipse?

Comment (rhamilto) [reply packages/patternfly-3/patternfly-react-extensions/less/vertical-tabs.less]: Yeah.  Offer option so users can decided on an instance basis? ` none | break-word | truncate`

**Comment by jeff-phillips-18** [comment]
Updated to offer wrap options via the `wrapStyle` attribute

**Pull request merged**